### ISSUE TITLE
Sometimes UI starts scrolling and jump by itself

### ISSUE THREAD

**Issue by matclab** [facundoolano/feedi#92]
First thank you for this nice idea and implementation !

Sometimes, when reading the posts, UI starts scrolling and jumping back and scrolling again. I guess it may be linked to RSS updates in background, but I don't know where to start to investigate about it.

I'll post some comment if I find something better to characterize the problem.

Version 906d8e059190cf14b363fbb11d99b51700a66280 hosted on fly.io.

**Comment by facundoolano** [comment]
Do you see this in the mobile or desktop UI or both? Let me know if you have some steps to reproduce.

> I guess it may be linked to RSS updates in background, but I don't know where to start to investigate about it.

There's no live updating of the feed so this shouldn't be the case.

The one scenario where I've perceived something like what you describe is when pining an entry: in that case the pins at the top get updated to include the new one and that causes the scroll position to 'jump'. I couldn't figure out how to tell the browser to "lock" on the currently visible entry to prevent that jump. I'm considering removing the live update in that case to prevent that artifact.

**Comment by matclab** [comment]
I've installed the last main before trying more debugging, but I've stumbled upon what's look like a DB which as not been migrated correctly:
```
sqlite3.OperationalError: no such column: entries.sort_date
```
Is there something special to do for migrations ?

**Comment by facundoolano** [comment]
Yes, you need to run `alembic upgrade head` from within the virtualenv. I use [this command](https://github.com/facundoolano/feedi/blob/HEAD/Makefile#L94-L101) to update the code, in case it helps.

**Comment by matclab** [comment]
Thanks, back to a working version.

For now, I've only seen the problem on desktop and I guess it is linked with the "mark view as read" functionality. 
I scroll to view some articles and if I come back, the titles are still blue (unread ?), I continue scrolling down and suddenly the view comes back to the article where I started. It has now a black title (read ?). I'll try to have a look at the sources this w-e (hopefully).

As a side note, it would be nice to have a "next button" to go to the next article without scrolling (which alleviates the eye strain due to scrolling).
